### PR TITLE
14.8.0

### PR DESCRIPTION
**Pull request by stephenjayakar**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    plaid (14.8.0)
+    plaid (14.7.0)
```
Comment (otherchen) [inline Gemfile.lock:4]: I think this should be 14.8.0

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 See full changelog for the OpenAPI schema (OAS) [here](https://github.com/plaid/plaid-openapi/blob/master/CHANGELOG.md).
 
 # 14.8.0
```
Comment (otherchen) [inline CHANGELOG.md:3]: should we be doing a new version bump instead?

Comment (stephenjayakar) [reply CHANGELOG.md:3]: like a major version?

Comment (stephenjayakar) [reply CHANGELOG.md:3]: i did a scan of the openapi changes and didn't think so?

File: Gemfile.lock (hunk 1 above)
Comment (stephenjayakar) [reply Gemfile.lock]: good catch!

File: CHANGELOG.md (hunk 2 above)
Comment (stephenjayakar) [reply CHANGELOG.md:3]: omg im stupid

**Pull request closed without merging**